src/components/layouts/Logo/index.tsx

```diff
@@ -3,19 +3,19 @@ import styled from '@emotion/styled';
 
 import __spacing from 'settings/__spacing';
 import { BRAND_PRIMARY, BRAND_WHITE } from 'settings/__color';
-import { fontWeights } from 'settings/__fonts';
+import { fontWeights, fontSizes } from 'settings/__fonts';
 import __layouts from 'settings/__layouts';
 
 interface LogoProps {
-    center?: boolean,
+  hasText?: boolean,
 }
 
 
-export default function Logo({center=false}: LogoProps) {
```

Comment:
Not sure why you removed this but removing this `center` prop from the logo actually breaks the ConfirmPassword page.

<img width="1440" alt="Screenshot 2019-12-28 at 10 08 51 PM" src="https://user-images.githubusercontent.com/38565349/71549497-cbec2f80-29be-11ea-853b-0b14c01422cd.png">

Reply:
Is this for mobile or desktop view? Why is the overlay image absent?
I think this is not in line with the designs. Can you look at them again on Figma? All the Authentication pages are alike.

Reply:
Just saw it now. Guess we would have to raise a bug fix for that one